Gopkg.toml

```diff
@@ -27,20 +27,20 @@
 
 [[constraint]]
   name = "github.com/coreos/etcd"
-  version = "3.3.10"
```

Comment:
why do we need this?

Reply:
@tnotb 

Why should we use '^'? if we download the latest version, then compatibility with the code will be broken, as for example in java it often happens